tests/Unit/Extension/CodeTransformExtra/Rpc/GenerateMethodHandlerTest.php

```diff
@@ -31,12 +36,17 @@ public function testProvidesOriginalSourceFromDiskIfPathIsNotTheGivenPath(): voi
     {
         $handler = $this->createHandler('generate_method');
         $source = SourceCode::fromStringAndPath(self::EXAMPLE_SOURCE, self::EXAMPLE_PATH);
-        $transformedSource = SourceCode::fromStringAndPath(self::EXAMPLE_SOURCE, __FILE__);
+        $thisFileContents = file_get_contents(__FILE__);
 
         $this->generateMethod->generateMethod(
             $source,
             self::EXAMPLE_OFFSET
-        )->willReturn($transformedSource);
+        )->willReturn(new TextDocumentEdits(
+            TextDocumentUri::fromString(__FILE__),
+            TextEdits::fromTextEdits([
+                TextEdit::create(strlen($thisFileContents) - 1, 1, substr($thisFileContents, -1) ."1")
```

Comment:
why "1" ?

Reply:
I wanted the change to just add "1" to the file contents, but it didn't work if I just described a text edit at `strlen(...)`, so I replaced the last character with itself + "1".